pkg/registry/graph.go

```diff
@@ -21,13 +21,13 @@ type Node interface {
 	Name() string
 	// Type returns the type of the registry element a Node refers to
 	Type() Type
-	// AncestorNames returns a set of strings containing the names of all of the node's ancestors
+	// Ancestors returns a set of strings containing the names of all of the node's ancestors
 	Ancestors() []Node
-	// DescendantNames returns a set of strings containing the names of all of the node's descendants
+	// Descendants returns a set of strings containing the names of all of the node's descendants
 	Descendants() []Node
-	// ParentNames returns a set of strings containing the names of all the node's parents
+	// Parents returns a set of strings containing the names of all the node's parents
 	Parents() []Node
-	// ChildrenNames returns a set of strings containing the names of all the node's children
+	// Childrens returns a set of strings containing the names of all the node's children
 	Childrens() []Node
```

Comment:
`Children`  ?

Reply:
I'll fix this in a followup if it's okay - I fixed up the godocs to silence a warning but changing unrelated interfaces in a feature PR is excessive.

Reply:
I extracted this to https://github.com/openshift/ci-tools/pull/1833